docs/types/conversion.rst

```diff
@@ -99,6 +99,26 @@ rules explicit::
     uint8 d = uint8(uint16(a)); // d will be 0x34
     uint8 e = uint8(bytes1(a)); // e will be 0x12
 
+``bytes`` arrays and ``bytes`` calldata slices can be converted to fixed bytes types(``bytes1``/.../``bytes32``).
+If array is longer than the target fixed bytes type panic will happen.
+If array is shorter than the target type, value will be padded on the right side.
```

Comment:
```suggestion
``bytes`` arrays and ``bytes`` calldata slices can be converted to fixed bytes types (``bytes1``/.../``bytes32``).
In case the array is longer than the target fixed bytes type, a panic will happen.
If the array is shorter than the target type, it will be padded with zeros on the right hand side.
```